diff --git a/src/mem_to_banks_detailed.sv b/src/mem_to_banks_detailed.sv
index 8255f00..6aa19f8 100644
--- a/src/mem_to_banks_detailed.sv
+++ b/src/mem_to_banks_detailed.sv
@@ -132,7 +132,7 @@ module mem_to_banks_detailed #(
     assign bank_req[i].wuser = wuser_i;
     assign bank_req[i].we    = we_i;
     stream_fifo #(
-      .FALL_THROUGH ( 1'b1         ),
+      .FALL_THROUGH ( 1'b0         ),
       .DATA_WIDTH   ( $bits(req_t) ),
       .DEPTH        ( FifoDepth    ),
       .T            ( req_t        )
